ambry-router/src/main/java/com/github/ambry/router/BackgroundDeleteRequest.java

```diff
@@ -24,6 +25,7 @@
   static final String SERVICE_ID_PREFIX = "ambry-background-delete-";
   private final StoreKey storeKey;
   private final String serviceId;
+  private final BlobId blobId;
```

Comment:
BlobId is a StoreKey (it implements the storeKey interface). So why introduce both and set one or the other? Can't storeKey be used in every case?